flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinator.java

```diff
@@ -1173,9 +1179,10 @@ public void startCheckpointScheduler() {
 			stopCheckpointScheduler();
 
 			periodicScheduling = true;
+			long initialDelay = schedulerInitialDelayGenerator.nextLong(
+				minPauseBetweenCheckpointsNanos / 1_000_000, baseInterval);
```

Comment:
Any place to check the value `minPauseBetweenCheckpointsNanos / 1_000_000` is less or equal than baseInterval?

Reply:
@yuqi1129 please see the constructor's [code segment](https://github.com/yanghua/flink/blob/1eb432833bf2dd23187194500b6e1c6523f30605/flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinator.java#L218)

Reply:
Could we replace `schedulerInitialDelayGenerator` with `long initialDelay = ThreadLocalRandom.current().nextLong(minPauseBetweenCheckpointsNanos / 1_000_000, baseInterval);`? That way we would not have to use `RandomUtils`.